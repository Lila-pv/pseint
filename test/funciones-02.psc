funcion fc1(b por copia)
    Escribir b
finfuncion

funcion a<-fc2(b por copia)
    a<-b
finfuncion

funcion fr1(b por referencia)
    Escribir b
finfuncion

funcion a<-fr2(b por referencia)
    b<-42
    a<-b
finfuncion

algoritmo prueba
    x<-1
    fc1(x) // ok
    fr1(x) // ok
    Escribir fc2(x) // ok
    Escribir fr2(x) // ok
    fc1(42) // ok
    fr1(42) // error
    Escribir fc2(42) // ok
    Escribir fr2(42) // error
finalgoritmo
