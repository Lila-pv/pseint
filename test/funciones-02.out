=== Line 25: SynError 268
=== Line 27: SynError 268
